apps/fluent-tester/src/FluentTester/TestComponents/Button/ButtonIconTest.tsx

```diff
@@ -13,48 +13,35 @@ const CustomizedIconButton = Button.customize({
 });
 
 export const ButtonIconTest: React.FunctionComponent<{}> = () => {
-  if (Platform.OS == ('win32' as any)) {
-    const testImage = require('./icon_24x24.png');
-    const testTtf = require('./Font Awesome 5 Free-Solid-900.otf');
+  const testImage = require('./icon_24x24.png');
 
-    const fontProps: FontIconProps = {
-      fontFamily: `Font Awesome 5 Free`,
-      fontSrcFile: testTtf,
-      codepoint: 0xf083,
-      fontSize: 16,
-    };
+  const svgProps: SvgIconProps = {
+    src: TestSvg,
+    viewBox: '0 0 500 500',
+  };
 
-    const svgProps: SvgIconProps = {
-      src: TestSvg,
-      viewBox: '0 0 500 500',
-    };
+  // SVG-based icons are not available on all platforms yet
+  const svgIconsEnabled = ['ios', 'macos', 'win32'].includes(Platform.OS as string);
 
-    return (
-      <View>
-        <Stack style={stackStyle}>
-          <Button icon={testImage} content="Button with png Icon" tooltip="button tooltip" />
-          <Button icon={{ fontSource: fontProps, color: 'blue' }} content="Button with font Icon" tooltip="button tooltip" />
```

Comment:
Since it's still a change to the win32 test app to remove this, I'd want someone from CXE to also look at this, so I'm just gonna tag @warren-ms . Things look good to me for the Apple side